articles/organizations/index.md

```diff
@@ -0,0 +1,122 @@
+---
+url: /organizations
+section: articles
+title: "Organizations - Beta"
+description: Learn about Auth0's Organizations feature, which is currently in beta.
+topics:
+  - organizations
+contentType: index
+useCase:
+  - build-an-app
+toc: true
+---
+
+# Organizations - Beta
+
+The Organizations feature represents a broad update to the Auth0 platform that allows our Business-to-Business (B2B) customers to better manage their partners and customers, and customize the ways that end-users in those organizations access their applications. Auth0 customers can use Organizations to:
+
+* manage the businesses that access their applications.
```

Comment:
Can we replace these bullets with: 

* Represent their business customers and partners in Auth0 and manage their membership.
* Configure branded, federated login flows for each business.
* Build administration capabilities into their products, using Organizations APIs, so that those businesses can manage their own organizations